boards/arm/stm32f3_disco/board.cmake

```diff
@@ -1,6 +1,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
 board_runner_args(jlink "--device=STM32F303VC" "--speed=4000")
+board_runner_args(stm32cubeprogrammer "--port=swd" "--reset=hw")
 
+include(${ZEPHYR_BASE}/boards/common/stm32cubeprogrammer.board.cmake)
```

Comment:
why is this added as the first runner? stm32cubeprogrammer does not allow e.g. to debug

Reply:
No particular reason, it was just for convenience on my own environment.
--> putting to the last entry